unishox2.c

```diff
@@ -147,32 +145,31 @@ unsigned int usx_mask[] = {0x80, 0xC0, 0xE0, 0xF0, 0xF8, 0xFC, 0xFE, 0xFF};
 /// Otherwise clen bits in code are appended to out starting with MSB
 int append_bits(char *out, int olen, int ol, byte code, int clen) {
 
-  byte cur_bit;
-  byte blen;
-  unsigned char a_byte;
-  int oidx;
 
   //printf("%d,%x,%d,%d\n", ol, code, clen, state);
 
   while (clen > 0) {
-     cur_bit = ol % 8;
```

Comment:
Is it possible to restore the indent?  It will help see the changes clearly.

Reply:
Fixed here.